src/js/components/action.jsx

```diff
@@ -17,6 +18,23 @@ export class ActionComponent extends Component {
         };
     }
 
+    onPostbackClick = () => {
+        this.setState({
+            state: 'processing'
+        });
+
+        postPostback(this.props._id).then(() => {
+            this.setState({
+                state: 'paid'
```

Comment:
Actually we should have specific states for the postback action if we do track state.